components/Payment/Form.js

```diff
@@ -112,7 +114,8 @@ const styles = {
   }),
   paymentMethodText: css({
     lineHeight: '40px',
-    verticalAlign: 'middle'
+    verticalAlign: 'middle',
+    color: colors.light.text
```

Comment:
```suggestion
```

Reply:
this is needed for the `rechnung` text on the third payment method tile (https://republik.ch/angebote?package=ABO_GIVE). Background is white and left to its own devices `rechnung`gets rendered in light grey in dark mode.

Reply:
got it!